backend/src/schema/resolvers/posts.js

```diff
@@ -211,10 +225,68 @@ export default {
       })
       return emoted
     },
+    pinPost: async (_parent, params, context, _resolveInfo) => {
+      let pinnedPost
+      const { driver, user } = context
+      const session = driver.session()
+      const { id: userId } = user
+      let writeTxResultPromise = session.writeTransaction(async transaction => {
+        const deletePreviousRelationsResponse = await transaction.run(
+          `
+          MATCH ()-[previousRelations:PINNED]->(post:Post)
+          DELETE previousRelations
+          RETURN post
+        `,
+          { params },
```

Comment:
`params` is not used here, as far I see …

Reply:
good catch, thanks